addons/account/views/account_move_views.xml

```diff
@@ -801,6 +801,8 @@
                                            attrs="{'invisible': [('type', 'not in', ('out_invoice', 'out_refund'))]}">
                                         <field name="company_id" groups="base.group_multi_company"/>
                                         <field name="invoice_user_id"/>
+                                        <field name="invoice_origin" string="Source Document"
+                                               attrs="{'invisible': [('type', '=', 'entry']}"/>
```

Comment:
@smetl is this ok or is there any type other than `out_invoice`, `out_refund`, `in_invoice`, `in_refund`, `out_receipt`, `in_receipt` ?

Reply:
`entry`